plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php

```diff
@@ -297,29 +297,19 @@ private static function order_received( $order_id = 0 ) {
 			return;
 		}
 
+		$template_args     = array( 'order' => $order );
```

Comment:
(1/3) Pretty minor, but I wonder if this should be:

```suggestion
		$template_args     = array( 
			'needs_verification' => false,
			'order'              => $order,
		);
```